optimole-wp.php

```diff
@@ -84,7 +84,8 @@ function optml() {
 	}
 
 	if ( ! defined( 'OPTML_DEBUG' ) ) {
-		define( 'OPTML_DEBUG', ( defined( 'TEST_GROUND' ) && TEST_GROUND ) ? true : false );
+		// define( 'OPTML_DEBUG', ( defined( 'TEST_GROUND' ) && TEST_GROUND ) ? true : false );
```

Comment:
We should revert to how it was before so we should uncomment this line and remove the one bellow